examples/client-cardano-database-v2/src/main.rs

```diff
@@ -39,6 +42,10 @@ pub struct Args {
     ancillary_verification_key: String,
 
     /// Aggregator endpoint URL.
+    ///
+    /// Either:
```

Comment:
Currently this comment is false since the code enforce direct url discovery, if you could share the logic from the client-cli allowing selection from the string then it would be true ! I made a comment in `mithril-client-cli/src/commands/mod.rs` that suggest a way to share that logic.